### PR TITLE
About page

### PR DESCRIPTION
**Pull request by Okewusi**
Adds the About page branch to the main branch

- The About link on the landing page leads to the About page 

- The About page has been completed and is responsive

- The page was built using the design mockup

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+</head>
+<body>
+    <div class="container">
+        <a class="home" href="./index.html"><h3 class="name">Okewusi Oluwapelumi</h3></a>
+        <p class="title">Front end developer and designer</p>
+        <div class="main-container">
+            <div class="about-me">
+                <h3>About Me</h3>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi autem commodi omnis deserunt, atque delectus ratione? Ad ipsum, quis esse corrupti ut accusamus odit quam natus. Obcaecati non cupiditate modi? Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt non sequi debitis corrupti nisi obcaecati accusamus quisquam saepe. Illo, eius animi accusamus vero quidem natus? Ratione sed laboriosam suscipit tempore.</p>
+            </div>
```
Comment (Monehin) [inline about.html]: Kindly come up with an write up that talks about you and your interest

File: about.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                            <p>Javascript</p>
+                            <p>React</p>
+                            <p>Mongodb</p>
+                        </div>
+                        <div class="right-skills">
+                            <p>HTML</p>
+                            <p>CSS</p>
+                            <p>Firebase</p>
+                        </div>
+                    </div>
```
Comment (Monehin) [inline about.html]: These list items are to represent and show your level of competence each skill. I can't seem to find your average percentage. Eg 
Html - 80% ±±±±=
CSS - 70%  ±±±==

File: about.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <div class="container">
+        <a class="home" href="./index.html"><h3 class="name">Okewusi Oluwapelumi</h3></a>
+        <p class="title">Front end developer and designer</p>
+        <div class="main-container">
+            <div class="about-me">
+                <h3>About Me</h3>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi autem commodi omnis deserunt, atque delectus ratione? Ad ipsum, quis esse corrupti ut accusamus odit quam natus. Obcaecati non cupiditate modi? Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt non sequi debitis corrupti nisi obcaecati accusamus quisquam saepe. Illo, eius animi accusamus vero quidem natus? Ratione sed laboriosam suscipit tempore.</p>
+            </div>
+            <div class="bottom">
+                <div class="picture"></div>
```
Comment (Monehin) [inline about.html]: kindly use your picture here

**Review by Monehin — CHANGES_REQUESTED** [review]
Why did you remove the social media icons from the home page?

File: about.html (hunk 1 above)
Comment (Okewusi) [reply about.html]: I didn't remove the icons, don't know why its that way. I will fix it.

File: about.html (hunk 2 above)
Comment (Okewusi) [reply about.html]: I didn't intend for them to show the average percentage, I just wanted to list the skills.

File: about.html (hunk 3 above)
Comment (Okewusi) [reply about.html]: I will put one random one, for now, I don't really have a professional one at the moment.

File: about.html (hunk 2 above)
Comment (Monehin) [reply about.html]: Ok noted!

**Pull request closed without merging**